AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -131,26 +142,136 @@ + (void)handleUrlResponse:(NSURL *)url {
   [MSALPublicClientApplication handleMSALResponse:url];
 }
 
-- (void)acquireToken {
-  if (self.clientApplication == nil) {
-    return;
++ (void)login {
+
+  // TODO protect with canBeUsed.
+  [[MSIdentity sharedInstance] login];
+}
+
+- (void)login {
+  @synchronized(lock) {
+    if (self.clientApplication == nil && self.identityConfig == nil) {
+      self.loginDelayed = YES;
+      return;
+    }
+    self.loginDelayed = NO;
+    [self.clientApplication acquireTokenForScopes:@[ (NSString * _Nonnull) self.identityConfig.identityScope ]
+                                  completionBlock:^(MSALResult *result, NSError *e) {
+                                    // TODO: Implement error handling.
+                                    if (e) {
```

Comment:
Why is there an empty block here?  Even id we're going to do error handling later I guess I would prefer to do `if (!e) {...}`

Reply:
Will be addressed in the next PR. Coming very soon.